src/loadgenerator/locustfile.py

```diff
@@ -188,10 +190,10 @@ def login(self):
                 found_token = False
                 for r_hist in response.history:
                     found_token |= r_hist.cookies.get('token') is not None
-                if found_token:
-                    response.success()
-                else:
-                    response.failure("login failed")
+                    if found_token:
+                        response.success()
+                    else:
+                        response.failure("login failed")
```

Comment:
Could you try maybe (more-or-less equivalent code but much simpler to read):
```
for r_hist in response.history:
    if r_hist.cookies.get('token') is not None:
        response.success()
response.failure("login failed")
```

Reply:
Ok, looks good, only will add `return` statement after `response.success()`.